docs/metrics-analysis-prometheus/collected-reported-metrics.md

```diff
@@ -0,0 +1,117 @@
+workflow job is `queued`, `in_progress`, or `completed`.
+
+## Workflow run
+
+Here are the different metrics collected by the GitHub Actions Exporter
+project for workflow runs:
+
+The number of workflow rebuilds: `github_actions_workflow_rebuild_count`.
+
+The duration of a workflow in seconds: `github_actions_workflow_duration_seconds`.
+
+Count the number of workflows for each state:
+
+- `github_actions_workflow_failure_count`
+- `github_actions_workflow_success_count`
+- `github_actions_workflow_cancelled_count`
+- `github_actions_workflow_inprogress_count`
+- `github_actions_workflow_total_count`
+
+## Workflow job
+
+Here are the different metrics collected by the GitHub Actions
+Exporter project for workflows and jobs.
+
+The duration of a job in seconds: `github_actions_job_duration_seconds`.
+
+Time between when a job is requested and started: `github_actions_job_start_duration_seconds`.
+
+Count the number of jobs for each states:
+
+- `github_actions_job_failure_count`
+- `github_actions_job_success_count`
+- `github_actions_job_cancelled_count`
+- `github_actions_job_inprogress_count`
+- `github_actions_job_queued_count`
+- `github_actions_job_total_count`
+
+## Cost metric
+
+This is the last metric we collect, and it is one of the most important
+ones. It allows us to determine the cost of our CI runs.
+
+### The formula to calculate the cost over a period of time
+
+To calculate this metric, we use the following formula:
+
+```bash
+cost = duration (per second) / 60 * cost (per minute)
+```
+
+### How do we find the cost per minute?
+
+#### GitHub
+
+As for GitHub, it is quite simple. They provide us with a fixed value, and
+the price never varies. To give an example, for `ubuntu-latest`, we have a cost
+of 0.008$/min, that's it. Easy!
+
+#### Self-Hosted
+
+When it comes to the cost of self-hosted runners, it's a bit more complicated.
+
+To calculate the costs of self-hosted runners, we can play the game of
+calculating for the main ones, namely AWS and Google Cloud Provider (GCP).
+
+The cost can be found based on the machine type in the Management Console
+for AWS (when creating an EC2 instance) and on the
+[Google Cloud website](https://cloud.google.com/compute/vm-instance-pricing)
+for GCP.
+
+Unfortunately, these values are not accurate as they lack several elements
+such as bandwidth or storage. As for storage costs, they can be found in
+the same places where the machine type cost is available. However, it is
+not possible to determine the bandwidth cost directly.
+
+To overcome this, we had to devise a workaround. We didn't necessarily
+need an exact cost for CI but rather a value close to reality (+/- 5%)
+for data visualization purposes.
+
+We analyzed previous invoices and calculated the additional cost generated
+by bandwidth, which amounted to approximately 30% for each month.
+Consequently, we were able to approximate the cost using the following formula:
```

Comment:
You're telling a story instead of documenting stuff, It's ok to write this in a blog post but documentation should get straight to the point. 

With the content you have here, define bullet points with the information that needs to be retrieved along with a short explanation.

Reply:
I tried something, tell me if it's better